os/kernel/init/os_bringup.c

```diff
@@ -252,6 +252,7 @@ static inline void os_do_appstart(void)
 	net_initialize();
 #endif
 
+#ifndef CONFIG_PREAPP_FROM_ELF
```

Comment:
I think we can use CONFIG_APP_SEPARATION instead of new config, CONFIG_PREAPP_FROM_ELF

Reply:
OK. done.